src/main/resources/templates/platforms.html

```diff
@@ -25,6 +25,13 @@ <h4>Select Your Game Platform</h4>
 				</a>
 			</div>
 		</div>
+		<br>
+		<div class="row">
+			<div class="col-sm">
+				<!-- Button directs to all games list -->
+				<a th:href="@{/game-list/list}" class="btn btn-info btn-lg"> All Games </a>
```

Comment:
For easier navigation, I added a button to the platforms home page that directs to the games list table.